zerver/tests/tests.py

```diff
@@ -1530,6 +1530,8 @@ def check_well_formed_change_settings_response(self, result):
         # type: (Dict[str, Any]) -> None
         self.assertIn("full_name", result)
 
+    # DEPRECATED, to be deleted after all uses of check_for_toggle_param
```

Comment:
Not needed https://chat.zulip.org/#narrow/near/102480/stream/GCI.20tasks/topic/testing.20toggle.20params